### DISCUSSION TITLE
Switch side of formation is missing?

### DISCUSSION THREAD

**Discussion by Cold-Burn** [ho-dev/HattrickOrganizer#1367] [General]
Switch side of formation is missing?
HO-5.0.3843.2-portable-win-JRE

**Comment by Cold-Burn** [comment]
It was only me that used this function?

**Comment by akasolace** [comment]
@wsbrenk I never used that feature .. was it removed on purpose ?

**Comment by wsbrenk** [comment]
![lineup4](https://user-images.githubusercontent.com/11821502/156945627-f5307765-6b9b-495c-b7e5-62e302e4d49a.jpg)

@akasolace lineup theming project removed the tagged side swap feature.

**Reply by Cold-Burn** [reply]
Can you add again? I used this frequently.
